boards/arm/olimexino_stm32/pinmux.c

```diff
@@ -57,13 +58,20 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA11, STM32F1_PINMUX_FUNC_PA11_USB_DM},
 	{STM32_PIN_PA12, STM32F1_PINMUX_FUNC_PA12_USB_DP},
 #endif /* CONFIG_USB_DC_STM32 */
+#ifdef CONFIG_CAN_1
+	{STM32_PIN_PB8, STM32F1_PINMUX_FUNC_PB8_CAN_RX},
+	{STM32_PIN_PB9, STM32F1_PINMUX_FUNC_PB9_CAN_TX},
+#endif /* CONFIG_CAN_1 */
 };
 
 static int pinmux_stm32_init(struct device *port)
 {
 	ARG_UNUSED(port);
 
 	stm32_setup_pins(pinconf, ARRAY_SIZE(pinconf));
+#ifdef CONFIG_CAN_1
+	AFIO->MAPR |= AFIO_MAPR_CAN_REMAP_REMAP2;
```

Comment:
Had overlook this line.
Can you add a comment stating this is due to this particular pin cofniguration ?

Reply:
Done